src/test/java/org/broadinstitute/hellbender/engine/dataflow/DataflowTestData.java

```diff
@@ -0,0 +1,215 @@
+package org.broadinstitute.hellbender.engine.dataflow;
+
+import com.google.cloud.dataflow.sdk.values.KV;
+import com.google.common.collect.Lists;
+import com.google.genomics.v1.Read;
+import htsjdk.samtools.SAMRecord;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.FakeReferenceSource;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.ReadContextData;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.ReferenceShard;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.VariantShard;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.reference.ReferenceBases;
+import org.broadinstitute.hellbender.utils.variant.SkeletonVariant;
+import org.broadinstitute.hellbender.utils.variant.Variant;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+public class DataflowTestData {
```

Comment:
This should either be renamed to `ReadsPreprocessingPipelineTestData` if it's specific enough that no one else will use it, or it should be moved somewhere like src/main/utils/test so that it's available to hellbender-protected.